### PR TITLE
cilicon 2.1.0 (new cask)

### PR DESCRIPTION
**Pull request by Marcocanc**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/cilicon.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+  url "https://github.com/traderepublic/Cilicon/releases/download/v#{version}/Cilicon.zip"
+  name "cilicon"
+  desc "Self-Hosted ephemeral CI on Apple Silicon"
+  homepage "https://github.com/traderepublic/Cilicon"
+
+  livecheck do
+    url :url
+    strategy :github_latest
+  end
```
Comment (SMillerDev) [inline Casks/cilicon.rb:13]: Are there ever pre-releases?

File: Casks/cilicon.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+  depends_on macos: :ventura
```
Comment (SMillerDev) [inline Casks/cilicon.rb:15]: I can't find this requirement anywhere upstream

File: Casks/cilicon.rb (hunk 1 above)
Comment (Marcocanc) [reply Casks/cilicon.rb:13]: We're not planning to have any. What would I need to adjust if there were?

Comment (SMillerDev) [reply Casks/cilicon.rb:13]: No, if you're not planning to have any this whole block is redundant

File: Casks/cilicon.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
+cask "cilicon" do
+  version "2.1.0"
+  sha256 "af39cff356afb5c22f006349a72f0bf1db02ad55bb2401eeb6bd360308fd9c01"
+
+  url "https://github.com/traderepublic/Cilicon/releases/download/v#{version}/Cilicon.zip"
+  name "cilicon"
```
Comment (bevanjkay) [inline Casks/cilicon.rb:6]:
```suggestion
  name "Cilicon"
```

File: Casks/cilicon.rb (hunk 1 above)
Comment (bevanjkay) [reply Casks/cilicon.rb:13]: @Marcocanc We can remove the whole block, as homebrew will default to using the `:git` strategy (inferred from the `url`) which doesn't rely upon the GitHub API.

File: Casks/cilicon.rb (hunk 2 above)
Comment (bevanjkay) [reply Casks/cilicon.rb:15]: @Marcocanc At the moment you are specifying that this runs only on Ventura. If Ventura is a minimum then the stanza should be adjusted to;

```suggestion ruby
  depends_on macos: ">= :ventura"
```

File: Casks/cilicon.rb (hunk 1 above)
Comment (Marcocanc) [reply Casks/cilicon.rb]: @bevanjkay thanks, appreciate the explanation!

**Review by bevanjkay — APPROVED** [review]

**Pull request merged**